src/ocean/task/util/StreamProcessor.d

```diff
@@ -184,6 +184,7 @@ class StreamProcessor ( TaskT : Task )
 
         if (throttler_config.resume_point == size_t.max)
             throttler_config.resume_point = total / 5;
+		else
```

Comment:
What about the `else` branch a few lines above? Was that also intended to not be always enforce?